Mathlib/Data/Nat/Digits.lean

```diff
@@ -729,7 +729,7 @@ theorem eleven_dvd_iff :
 
 theorem eleven_dvd_of_palindrome (p : (digits 10 n).Palindrome) (h : Even (digits 10 n).length) :
     11 ∣ n := by
-  let dig := (digits 10 n).map fun n : ℕ => (n : ℤ)
+  let dig := (digits 10 n).map (fun i : ℕ => (i : ℤ))
```

Comment:
(why) do we need this change?

Reply:
Apparently not, reverted. (And cherry-picked back to nightly-testing)